extensions/brew/src/brew.ts

```diff
@@ -100,7 +100,11 @@ export interface OutdatedResults {
 
 /// Paths
 
+export const customBrewPath =
+  preferences.customBrewPath && preferences.customBrewPath.length > 0 ? preferences.customBrewPath : undefined;
+
 export const brewPrefix: string = (() => {
+  if (customBrewPath) return path_join(customBrewPath, "..", "..");
```

Comment:
I think something like this would potentially be more robust and future proof:

```javascript
const brew = if (customBrewPath) ? customBrewPath : "brew";
try {
  return execSync(`${brew} --prefix`, { encoding: "utf8" }).trim();
} catch { ... }

Reply:
Good point.

In that case I'd suggest this:

```typescript
export const brewPrefix: string = (() => {
  const customBrewPath = preferences.customBrewPath && preferences.customBrewPath.length > 0 ? preferences.customBrewPath : undefined;
  if (customBrewPath) {
    return execSync(`${customBrewPath} --prefix`, { encoding: "utf8" }).trim();
  }
  try {
    return execSync("brew --prefix", { encoding: "utf8" }).trim();
  } catch { ... }
}
```

Reply:
In retrospect it's a shame that the custom path preference did not specify the brew directory (e.g. `/opt/homebrew`) and not the binary path. But it's a bit late to change that now 😢